lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -938,10 +938,24 @@ EmitInfo WasmBytecodeGenerator::EmitLoop()
     Js::ByteCodeLabel loopHeadLabel = m_writer->DefineLabel();
     Js::ByteCodeLabel loopLandingPadLabel = m_writer->DefineLabel();
 
-    uint32 loopId = m_writer->EnterLoop(loopHeadLabel);
+    // Push possibly yielding loop label before capturing all the yielding registers
+    BlockInfo implicitBlockInfo = PushLabel(loopTailLabel);
+
+    // Save all the possible blocks the loop could yield to
```

Comment:
maybe change this comment? You are saving all the registers that are internal to the loop, there's nothing really about blocks

Reply:
Right, I forgot to update the comment in my last iteration thanks